subiquity/server/apt.py

```diff
@@ -182,7 +183,7 @@ async def apply_apt_config(self, context):
 
         await run_curtin_command(
             self.app, context, 'apt-config', '-t', self.configured_tree.p(),
-            config=config_location)
+            config=config_location, private_mounts=False)
```

Comment:
Why is this private_mounts=False? I *think* this is one of the ones we really want to be isolated, isn't it?

Reply:
Yes, you are right.
In the first place I set `private_mounts=True` almost everywhere and obviously it didn't work because some calls are expected to leave some mounts available for later (e.g. /target).
So I ended-up do it only for the `curtin in-target` commands. But obviously, `curtin apt-config` should use private mounts too. Thanks!

Reply:
Updated!